flavour.json

```diff
@@ -0,0 +1,33 @@
+{
+  "fS": 9.22e9,
+  "data": {
```

Comment:
Have you decided yet on how to handle upper/lowercase and various string codings?